packages/react-router/tsconfig.legacy.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
What's the purpose of this file?

Reply:
It only runs type tests on the src files for TS 5.0-5.4, while TS 5.5 tests src and tests. This cuts down the task's time dramatically.